### PR TITLE
Adding Grafana dashboard for SLO/SLI

### PR DESCRIPTION
**Pull request by dislbenn**
Issues: https://github.com/open-cluster-management/backlog/issues/14849 & https://github.com/open-cluster-management/backlog/issues/14851.

### CODE REVIEW

**Comment by marcolan018** [comment]
the file names should be included in kustomization.yaml

**Comment by dislbenn** [comment]
> the file names should be included in kustomization.yaml

Added file name to `kustomization.yaml`.

File: operators/multiclusterobservability/manifests/base/grafana/kustomization.yaml [hunk 1]
```diff
@@ -17,5 +17,7 @@ resources:
 - dash-k8s-compute-resources-workload.yaml
 - dash-k8s-compute-resources-node-pods.yaml
 - dash-k8s-compute-resources-pod.yaml
+# - dash-k8s-service-level-overview-api-server.yaml
```
Comment (dislbenn) [inline operators/multiclusterobservability/manifests/base/grafana/kustomization.yaml:20]: Adding this file next sprint.

**Pull request merged**